src/cpu/aarch64/injectors/jit_uni_eltwise_injector.hpp

```diff
@@ -215,12 +222,16 @@ struct jit_uni_eltwise_injector_f32_t {
     void exp_compute_vector_fwd(const TRegS &vmm_src);
     void relu_compute_vector_fwd(const TRegS &vmm_src);
     void relu_zero_ns_compute_vector_fwd(const TRegS &vmm_src);
+    void relu_zero_ns_compute_vector_fwd(const TRegH &vmm_src);
     void elu_compute_vector_fwd(const TRegS &vmm_src);
     void tanh_compute_vector_fwd(const TRegS &vmm_src);
     void tanh_polynomial_approx_compute_vector_fwd(const TRegS &vmm_src);
     void square_compute_vector_fwd(const TRegS &vmm_src);
+    void square_compute_vector_fwd(const TRegH &vmm_src);
```

Comment:
I like the way you have used overloading here, although if we had lots more of these, we could consider using templates, because the logic is identical

Reply:
Makes sense, I like the way it is now!